data/json/items/comestibles.json

```diff
@@ -3201,10 +3189,10 @@
         "nutrition" : 6,
         "description" : "A smooth grayish block of cleaned and rendered monster fat.  It will remain 'fresh' for a very long time, and can be used as an ingredient in many projects.  You could eat it, but it will poison you.",
         "price" : 0, "//" : "Not for use in edible/foodsafe recipes.  Inefficiency is handled by tainted tallow recipe requiring 200% as much materials input as regular tallow recipe.",
-        "material" : "junk",
+        "material" : "flesh",
```

Comment:
Flesh is fine here since:
* It can be consumed by carnivores
* It can't be consumed by herbivores
* Oil material wouldn't fix much regarding vitamins (vegetable oil would need different vitamins anyway), meaning vitamins would have to be specified explicitly anyway
* There are no other important material-derived properties used here